cli/pyconv.ts

```diff
@@ -1443,7 +1443,6 @@ let funMap: Map<FunOverride> = {
     "ustruct.calcsize": { n: "pins.packedSize", t: tpNumber },
     "pins.I2CDevice.read_into": { n: ".readInto", t: tpVoid },
     "bool": { n: "!!", t: tpBoolean },
-    "Array.index": { n: ".indexOf", t: tpNumber },
```

Comment:
I see that this is a mapping of TS to Python, but curious what "n" and "t" stand for in this object?

Reply:
well `t` is definitely type, not sure what the etymology of `n` is but it's the text that gets replaced. bad names i agree